Lob-API-public.yml

```diff
@@ -34,12 +56,58 @@ tags:
   - name: Bank Accounts
     description: Bank Accounts allow you to store your bank account securely in our system.
 
+  - name: Beta Program
+    description: |
+      Here at Lob, we pride ourselves on building high quality platform capabilities rapidly 
+      and iteratively, so we can constantly be delivering additional value to our customers. 
+      When evaluating a new product or feature from Lob, you may see that it has been released in Beta.
+
+      Typically, something in Beta means that the feature is early in its lifecycle here at 
+      Lob. While we fully stand behind the quality of everything we release in Beta, we do 
+      anticipate receiving a higher level of customer feedback on Beta features, as well as a 
+      faster pace of changes from our engineering team in response to that feedback.
+
+      By participating in a Lob Beta program, you will have the opportunity to get early access 
+      to a new product capability, as well as having a unique opportunity to influence the product's 
+      direction with your feedback.
+
+      You should also anticipate that features in Beta may have functional or design limitations, 
+      and might change rapidly as we receive customer feedback and make improvements. In particular, 
+      new APIs in Beta may also go through more frequent versioning and version deprecation cycles 
+      than our more mature APIs.
+
+      If you are participating in a Beta program and want to provide feedback, please feel free to 
+      [contact us](https://lob.com/support#contact)!
+
+      **Current Beta Features** 
```

Comment:
Let's remove the list of current beta features, since it is wildly out of date. Instead, we should look for some way to mark features as beta dynamically in the menu, similarly to what is done with the current docs.